app/src/main/res/raw/about.html

```diff
@@ -42,7 +42,13 @@
 <html>
     <body
         class="about">
-        <h1 id="wordmark">Fenix</h1>
+        <br>
+        <center>
+            <img src="https://user-images.githubusercontent.com/4400286/52872955-a33eac80-3102-11e9-84b8-9c47efcb7ee3.png"
```

Comment:
I recognize loading an image from a remote location is not ideal for a statically served page. I tried for a while to use various methods of loading a local resource file (through resources like [this](https://stackoverflow.com/a/5856258)) but was not able to get it working. If someone has any ideas on how to improve feel free to chime in!

Reply:
You can always load from Android assets.